src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -116,11 +118,12 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         Class updatedClassDateTime = editPersonDescriptor.getAClass().orElse(personToEdit.getAClass());
         Money updatedMoneyOwed = editPersonDescriptor.getMoneyOwed().orElse(personToEdit.getMoneyOwed());
         Money updatedMoneyPaid = editPersonDescriptor.getMoneyPaid().orElse(personToEdit.getMoneyPaid());
+        Money updatedMoneyPerClass = editPersonDescriptor.getMoneyPerClass().orElse(personToEdit.getMoneyPerClass());
```

Comment:
I think updatedRatesPerClass and getRatesPerClass sounds more natural

Reply:
Noted! Will do in the subsequent commit ;)